torch/csrc/jit/interpreter.cpp

```diff
@@ -400,6 +400,10 @@ struct CodeImpl {
     return instructions_;
   }
 
+  const std::vector<Node*>& instructions_source() const {
```

Comment:
If we have instructions_source then we probably do not need opname_table since the operator name can be extracted from the Node.